src/preset.js

```diff
@@ -7,26 +7,81 @@
 /** @const */
 genotet.preset = {};
 
+/** @private @const {number} */
+genotet.preset.DEFAULT_TRACK_NUMBER_ = 1;
+/** @private @const {number} */
+genotet.preset.BINDING_TRACK_NUMBER_ = 3;
+
 /**
  * Loads a preset with the given name.
  * @param {string} preset
  */
 genotet.preset.loadPreset = function(preset) {
+  genotet.viewManager.closeAllViews();
+
   if (!preset) {
     preset = 'default';
   }
-
   switch (preset) {
     case 'default':
+      genotet.preset.createExpression_();
+      genotet.preset.createNetwork_();
+      genotet.preset.createBinding_(genotet.preset.DEFAULT_TRACK_NUMBER_);
       break;
     case 'network':
+      genotet.preset.createNetwork_();
+      genotet.preset.createBinding_(genotet.preset.DEFAULT_TRACK_NUMBER_);
       break;
     case 'expression':
+      genotet.preset.createExpression_();
+      genotet.preset.createNetwork_();
       break;
     case 'binding':
+      genotet.preset.createExpression_();
+      genotet.preset.createNetwork_();
+      genotet.preset.createBinding_(genotet.preset.BINDING_TRACK_NUMBER_);
       break;
     default:
       genotet.error('unknown preset:', preset);
       return;
   }
 };
+
+/**
+ * Create a preset expression view.
+ * @private
+ */
+genotet.preset.createExpression_ = function() {
+  genotet.viewManager.createView('expression', 'My Expression Matrix', {
+    fileName: 'expressionMatrix',
```

Comment:
Ok. I haven't read too much datasets yet. Shall we pick up some good set of example datasets together?